setup.py

```diff
@@ -17,7 +17,15 @@
 
 if os.environ.get('USE_NCCL', '1') == '1':
     cxx_flags.append('-DFMOE_USE_NCCL')
-    ext_libs.append('nccl')
+    if os.environ.get('USE_ROCM', '0') == '1':
```

Comment:
Maybe you could extract ROCM detection to a variable to avoid duplicate code.

Reply:
ok.

Reply:
Extract the ROCM detection from pytorch to `is_rocm_pytorch` variable.